### PR TITLE
add an overview for end user guide

### PR DESCRIPTION
**Pull request by wonderflow**
ref to https://github.com/oam-dev/kubevela/issues/1403

### CODE REVIEW

File: docs/en/end-user/explore.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+title: Explore Application
```
Comment (resouer) [inline docs/en/end-user/explore.md:2]: Manage Application

File: docs/en/end-user/expose.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+title: Expose Application Publicly
```
Comment (resouer) [inline docs/en/end-user/expose.md:2]: Expose Application

File: docs/en/end-user/expose.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+---
+
+TBD, Content Overview：
+
+1. add service/ingress/route trait.
```
Comment (resouer) [inline docs/en/end-user/expose.md:7]: Please choose from svc ingress route.

File: docs/en/application.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 ---
-title:  Application CRD
+title: Deploy Application
```
Comment (resouer) [inline docs/en/application.md:2]: This doc need rewrite, it should only about deploy, no definitions and "contracts" should be included.

File: docs/en/end-user/monitoring.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+---
+title: Monitoring
+---
+
+TBD, Content Overview：
+
+1. Install monitoring trait(along with prometheus/grafana controller).
```
Comment (resouer) [inline docs/en/end-user/monitoring.md:7]: Move all installation scripts to a separate doc may be named `Install Capability Providers` (e.g. https://knative.dev/docs/install/install-extensions/)

File: docs/en/end-user/patch.md [hunk 6]
```diff
@@ -0,0 +1,9 @@
+---
+title: Patch Application
```
Comment (resouer) [inline docs/en/end-user/patch.md:2]: Users never care about "Patch".

This should be sth like "Attach Sidecar" or other specific features, and these features should be built-in.

**Comment by resouer** [comment]
I updated the sidebar example a bit: https://github.com/oam-dev/kubevela/issues/1403#issuecomment-812091766, since we now plan to have user guides, we could make the sidebar more clear.

- User Guide - how to deploy and rollout app
- Component Provider Guide - how to add more components in the system
- Traits Management Guide - how to add more operational features in the system

File: docs/en/end-user/expose.md (hunk 3 above)
Comment (wonderflow) [reply docs/en/end-user/expose.md:7]: I would like to choose ingress if only one

**Review by zzxwill — APPROVED** [review]
/lgtm

**Pull request merged**